README.md: switch to https for IAIS maven repository and reflect chan… (#81)
IndustrialDataSpace/InformationModel: README.md

## Related resources

> Additional resources are now provided for everyone (formerly restricted to International Data Spaces Association members):

Reviewer: ```suggestion
Additional resources are provided:
```

</dependency>
```

> - UML-based visualization of the Information Model, its classes, properties and relations, including their cardinalities. It is available in the Nightly and Release Packages on [GitHub](https://github.com/IndustrialDataSpace/InformationModel/releases).

Reviewer: ```suggestion
- UML-based visualization of the Information Model, its classes, properties and relations, including their cardinalities. It is available as part of the Nightly and Release Packages on [GitHub](https://github.com/IndustrialDataSpace/InformationModel/releases).
```

For members of the International Data Spaces Association only:

> - Working [document](https://industrialdataspace.jiveon.com/docs/DOC-1817), on the association's Jive platform, explaining

Reviewer: ```suggestion
- Working [document](https://industrialdataspace.jiveon.com/docs/DOC-1817), explaining
```

> the usage of the Information Model for communication on the Industrial Data Space.

Reviewer: ```suggestion
the usage of the Information Model for data exchanges in the IDS.```
